src/index.ts

```diff
@@ -1,3 +1,32 @@
-import { DynamoDB } from './lib/dynamodb';
+import { DynamoDB } from './lib';
 
-export = new DynamoDB();
+export {
+	ReadItem,
+	TransactRead,
+	TransactQuery,
+	TransactDeleteItem,
+	TransactInsertItem,
+	TransactUpdateItem,
+	WriteItem,
+	TransactWrite,
+	TransactMethod,
+	BaseQuery,
+	CreateTable,
+	DeleteItem,
+	DeleteTable,
+	Executable,
+	InsertItem,
+	ListTables,
+	Method,
+	Query,
+	Scan,
+	UpdateItem,
+	Map,
```

Comment:
Where is this one used? I don't really like how I named this one as it's a colission with the JavaScript `Map`. So if we could keep it internal, I would love to do that because I think I want to rename this one.

Reply:
It is used for `ExpressionAttributeNames`, `ExpressionAttributeValues` and `Table.insert()`